packages/@react-aria/interactions/stories/styles.css

```diff
@@ -0,0 +1,8 @@
+body {
+  height: 50px !important;
+  width: 50px !important;
```

Comment:
Looks like these styles are being applied to the body of all stories, not just the useInteractOutside story. Lemme see if I can figure out why

Reply:
huh... that's annoying, thanks for catching that
we can probably make a component that applies these styles on mount and removes them on unmount, then just decorate this story with that